packages/framework/src/Resources/config/services.yml

```diff
@@ -13,6 +13,15 @@ services:
     autoconfigure: true
     public: false
 
+  Shopsys\FrameworkBundle\:
+      resource: '../../{Component,Model,DataFixtures}/**/{*Generator.php,*Resolver.php,*Provider.php,*Registry.php,*Factory.php,*Service.php,*Facade.php,*Creator.php,*Repository.php,*Mapper.php,*Listener.php,*Subscriber.php,*Loader.php,*Injector.php,*Reader.php,*Writer.php,*Setting.php,*Config.php}'
+
+  Shopsys\FrameworkBundle\Component\:
+    resource: '../../Component/**/{*Transformer.php,*Finder.php,*Parser.php,*Comparator.php,*Quoter.php,*CompilerPass.php}'
+
+  Shopsys\FrameworkBundle\Model\:
+    resource: '../../Model/**/{*Scheduler.php,*Handler.php,*Calculation.php,*Recalculator.php,*Filter.php,*Mail.php,*InlineEdit.php}'
```

Comment:
I would rather auto-discover classes with these prefixes in the whole `src` directory so it's simpler to use and maintain. Why did you separate it into 3 sets?

Reply:
These sets seem clearer for me. Do you want to merge that?